guides/v2.2/javascript-dev-guide/widgets/widget-folder-tree.md

```diff
@@ -0,0 +1,148 @@
+---
+group: javascript-developer-guide
+title: FolderTree widget
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The FolderTree [widget](https://glossary.magento.com/widget/) allows creating a dynamic hierarchical tree structure with folders.
+
```

Comment:
Please explain what this widget does in more detail. What can you do with it? What is the point of a dynamic hierarchical tree structure?